modular_skyrat/master_files/code/modules/client/preferences/mutant_parts.dm

```diff
@@ -0,0 +1,1292 @@
+	target.dna.mutant_bodyparts["moth_markings"]["color"] = list(sanitize_hexcolor(value[1]), sanitize_hexcolor(value[2]), sanitize_hexcolor(value[3]))
+
+/// Fluff
+
+/datum/preference/toggle/fluff
+	category = PREFERENCE_CATEGORY_SECONDARY_FEATURES
+	savefile_identifier = PREFERENCE_CHARACTER
+	savefile_key = "fluff_toggle"
+	relevant_mutant_bodypart = "fluff"
+	default_value = FALSE
+
+/datum/preference/toggle/fluff/apply_to_human(mob/living/carbon/human/target, value, datum/preferences/preferecnes)
+	return TRUE // we dont actually want this to do anything
+
+/datum/preference/toggle/fluff/is_accessible(datum/preferences/preferences)
+	var/passed_initial_check = ..(preferences)
+	var/allowed = preferences.read_preference(/datum/preference/toggle/allow_mismatched_parts)
+	return passed_initial_check || allowed
+
+/datum/preference/choiced/fluff
+	savefile_key = "feature_fluff"
+	savefile_identifier = PREFERENCE_CHARACTER
+	category = PREFERENCE_CATEGORY_SECONDARY_FEATURES
+	relevant_mutant_bodypart = "fluff"
+
+/datum/preference/choiced/fluff/is_accessible(datum/preferences/preferences)
+	var/passed_initial_check = ..(preferences)
+	var/allowed = preferences.read_preference(/datum/preference/toggle/allow_mismatched_parts)
+	var/part_enabled = preferences.read_preference(/datum/preference/toggle/fluff)
+	return ((passed_initial_check || allowed) && part_enabled)
+
+/datum/preference/choiced/fluff/init_possible_values()
+	return assoc_to_keys(GLOB.sprite_accessories["fluff"])
+
+/datum/preference/choiced/fluff/apply_to_human(mob/living/carbon/human/target, value)
+	if(!target.dna.mutant_bodyparts["fluff"])
+		target.dna.mutant_bodyparts["fluff"] = list("name" = "None", "color" = list("FFF", "FFF", "FFF"))
+	target.dna.mutant_bodyparts["fluff"]["name"] = value
+
+/datum/preference/choiced/fluff/create_default_value()
+	var/datum/sprite_accessory/fluff/moth/none/default = /datum/sprite_accessory/fluff/moth/none
+	return initial(default.name)
+
+/datum/preference/tri_color/fluff
+	category = PREFERENCE_CATEGORY_SECONDARY_FEATURES
+	savefile_identifier = PREFERENCE_CHARACTER
+	savefile_key = "fluff_color"
+	relevant_mutant_bodypart = "fluff"
+
+/datum/preference/tri_color/fluff/is_accessible(datum/preferences/preferences)
+	var/passed_initial_check = ..(preferences)
+	var/allowed = preferences.read_preference(/datum/preference/toggle/allow_mismatched_parts)
+	var/part_enabled = preferences.read_preference(/datum/preference/toggle/fluff)
+	return ((passed_initial_check || allowed) && part_enabled)
+
+/datum/preference/tri_color/fluff/apply_to_human(mob/living/carbon/human/target, value)
+	if(!target.dna.mutant_bodyparts["fluff"])
+		target.dna.mutant_bodyparts["fluff"] = list("name" = "None", "color" = list("FFF", "FFF", "FFF"))
+	target.dna.mutant_bodyparts["fluff"]["color"] = list(sanitize_hexcolor(value[1]), sanitize_hexcolor(value[2]), sanitize_hexcolor(value[3]))
+
+/// Head Accessories
+
+/datum/preference/toggle/head_acc
```

Comment:
This should probably be renamed to `/datum/preference/toggle/head_accessories`, there's no need to make it shorter here.